server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -141,6 +141,8 @@ private void createFateTable(ServerContext context) {
     } catch (Exception e) {
```

Comment:
This is a very broad catch block. It seems like we are looking for a very specific stack trace here. This catch block could be made more narrow to just check for the type(s) we are actually looking for that would have this as a cause in this situation.